_readings/feminism-without-borders.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: feminism-without-borders
+title: "Feminism Without Borders"
+authors: "Chandra Talpade Mohanty"
+tags: feminism anti-racism anti-capitalism
```

Comment:
I like `anti-racism` better, because it's more descriptive in terms of the book's perspective about race. Not that we'd ever be "pro-racism," but "race" as a category feels incredibly broad/could encompass some weird things.